blog/2019-06-06-why-should-we-be-present-at-events.md

```diff
@@ -0,0 +1,86 @@
+
+I handed a console to David (a developer from the InfluxDB community) and asked him nicely if he would review our implementation and configuration of InfluxDB. 
+5 minutes later there was a warm applause because we – the openHAB-Community – did everything right. In short: InfluxDB inspected our implementation and gave us the confidence of using InfluxDB the right way. Check. **Thanks David for investing your time**!
+
+At the next opportunity I am determined to talk to community members of the other open source software that we use (e. g. Grafana, Node-RED …) And I found a label for that special kind of communication: *cross-community communication*. We – as open source communities – should feel a strong urge to share our experience and help each other. 
+
+A second highlight of the Hackathon event was a 20 minute talk to a Bosch guy. He introduced himself as **the biggest fan of our project**. 
+I‘ve never heard from him. He told me that he has no time to be part of our community. But he really likes the strong and sharing spirit of our community. Later on he gave me his business card and told me if we ever need help to address the right people in the Bosch universe he wants to help me finding the right person. Check! [target groups 2 and 3]
+
+Another vibrant event immediately followed the really cool Bosch Hackathon - I attended the 
+
+## Maker Faire Berlin. 
+Attendees: the do-it-yourself (DYI) community. Starting on Friday with a day exclusively for school children. That is one of our target groups. The young talents. We need them [target group 1] because many of us are "really old" and as a sustainable open source software community we have to hand the project over to the younger ones. You can imagine the fun with young people if you show them the smart home. They spontaneously understand the dependencies.
+
+Saturday and Sunday were open for general public. I’ve never met so many happy people, **giving a warm applause and a big thank you** to me (as the representative of the openHAB community) **for our software project**. Most of them were never active in our forum. I invited them to switch their habits and be part of our wonderful community. „Say 'we' instead of 'you' please!“ [target group 2]
+
+On Sunday I gave a session in the venue cinema with the topic “What is the 'smart' in smart home?”. [all target groups]
+
+After 7 days in a row of supporting our fantastic project openHAB, I felt a bit like never ever talking about home automation again :) 
+One week later I attend the Smart Home Day in Darmstadt ... But that is another story.
+
+My call to action for our community: Keep in mind (at every occasion) that we have marketing goals. Let's turn interested people into amateurs and turn amateurs into community members. Let's grow the community! 
+
+I want to thank the people who helped making these events happen. Take a look at their faces on twitter. 
+https://twitter.com/luckow/status/1129754848474423298<br />
```

Comment:
Turn that into a real list maybe? A small twitter emblem before each link would be sweet as.

Reply:
check. (sorry without twitter emblem)